tests/db_functions/math/test_degrees.py

```diff
@@ -0,0 +1,44 @@
+import math
+from decimal import Decimal
+
+from django.db.models import DecimalField
+from django.db.models.functions import Degrees
+from django.test import TestCase
+
+from ..models import DecimalModel, FloatModel, IntegerModel
+
+
+class DegreesTests(TestCase):
+
+    def test_decimal(self):
+        DecimalModel.objects.create(n1=Decimal('-12.9'), n2=Decimal('0.6'))
+        obj = DecimalModel.objects.annotate(n1_d=Degrees('n1'), n2_d=Degrees('n2')).first()
+        self.assertAlmostEqual(obj.n1_d, math.degrees(obj.n1))
+        self.assertAlmostEqual(obj.n2_d, math.degrees(obj.n2))
+
+    def test_float(self):
+        FloatModel.objects.create(f1=-27.5, f2=0.33)
+        obj = FloatModel.objects.annotate(f1_d=Degrees('f1'), f2_d=Degrees('f2')).first()
+        self.assertAlmostEqual(float(obj.f1_d), math.degrees(obj.f1))
+        self.assertAlmostEqual(float(obj.f2_d), math.degrees(obj.f2))
```

Comment:
Remove the cast to `float` for these - it shouldn't be necessary due to `output_field = FloatField()`.